.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,4 +1,4 @@
-name: Report a bug with the UI, Frontend or Lovelace
+name: Report a bug with the Frontend / User Interface
```

Comment:
```suggestion
name: Report a bug with the Frontend / User Interface / Dashboards
```

Reply:
Isn't that a bit redundant? I added "User Interface" since users might not feel comfortable with "Frontend" (rather tech-y term).

But I wold also not veto adding dashboards in as well.

Reply:
I would remove frontend and just have UI/Dashboards

Frontend makes everything redundant haha

UI is everything as well but dashboards just reassures newer users they are in the right place

Reply:
I am fine with that. PR updated.